frontend/src/components/pages/admin/QuestionPage.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { DndProvider } from "react-dnd";
+import { HTML5Backend } from "react-dnd-html5-backend";
+import { Flex } from "@chakra-ui/react";
+
+import { HOME_PAGE } from "../../../constants/Routes";
+import QuestionSidebar from "../../question-creation/QuestionSidebar";
+import QuestionEditor from "../../question-creation/QuestionEditor";
+
+import { QuestionElement } from "../../../types/QuestionTypes";
+
+const QuestionPage = (): React.ReactElement => {
+  const [questionElements, setQuestionElements] = React.useState<
+    QuestionElement[]
+  >([]);
+
+  const addQuestionElement = (newQuestionElement: QuestionElement) => {
+    setQuestionElements((prevQuestionElements) => [
+      ...prevQuestionElements,
+      newQuestionElement,
+    ]);
+  };
```

Comment:
I think in the next PR, I'm going to refactor this to use `useContext` to avoid prop drilling + for a cleaner implementation. Just wanted to get this PR merged since the ticket has been open for waaaaaay too long!